packages/p2p/src/components/my-ads/create-ad-form.jsx

```diff
@@ -17,22 +17,23 @@ import { observer } from 'mobx-react-lite';
 import { Localize, localize } from 'Components/i18next';
 import { useUpdatingAvailableBalance } from 'Components/hooks';
 import { buy_sell } from 'Constants/buy-sell';
+import { ad_type } from 'Constants/floating-rate';
 import { useStores } from 'Stores';
 import CreateAdSummary from './create-ad-summary.jsx';
 import CreateAdErrorModal from './create-ad-error-modal.jsx';
 import CreateAdFormPaymentMethods from './create-ad-form-payment-methods.jsx';
 import CreateAdAddPaymentMethodModal from './create-ad-add-payment-method-modal.jsx';
+import FloatingRate from '../floating-rate';
```

Comment:
```suggestion
import FloatingRate from '../floating-rate.jsx';
```

Reply:
Since I am making use of a barrel file (index.js) present under floating-rate directory, we can ignore the extension

Reply:
Yes, it's possible to ignore it of course, but this is just a suggestion to follow our code standarts since we add code extensions across all other imports here